packages/filesystem/src/node/download/file-download-collection.ts

```diff
@@ -0,0 +1,89 @@
+/********************************************************************************
+ * Copyright (C) 2019 Bitsler and others.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License v. 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0.
+ *
+ * This Source Code may also be made available under the following Secondary
+ * Licenses when the conditions for such availability set forth in the Eclipse
+ * Public License v. 2.0 are satisfied: GNU General Public License, version 2
+ * with the GNU Classpath Exception which is available at
+ * https://www.gnu.org/software/classpath/license.html.
+ *
+ * SPDX-License-Identifier: EPL-2.0 OR GPL-2.0 WITH Classpath-exception-2.0
+ ********************************************************************************/
+import { injectable, inject } from 'inversify';
+import { ILogger } from '@theia/core/lib/common/logger';
+import * as rimraf from 'rimraf';
+
+export interface DownloadStorageItem {
+    file: string;
+    root?: string;
+    size: number;
+    remove: boolean;
+    expire?: number;
+}
+
+@injectable()
+export class FileDownloadCollection {
+
+    @inject(ILogger)
+    protected readonly logger: ILogger;
+
+    protected readonly downloads = new Map<string, DownloadStorageItem>();
+    protected readonly expireTime: number = 1;
```

Comment:
Please add a `protected` method instead. The `expireTime` name should refect, it is in minutes:
```js
protected expireTimeInMinutes(): number {
  return 1;
}
```